iroh-net/src/magicsock/node_map/best_addr.rs

```diff
@@ -141,6 +143,22 @@ impl BestAddr {
         }
     }
 
+    /// Reset the expiry, if the passed in addr matches the currently used one.
+    pub fn reconfirm_if_used(&mut self, addr: SocketAddr, source: Source, confirmed_at: Instant) {
+        match self.0.as_mut() {
+            None => {
+                // Nothing to do
+                // TODO: do we want to store this regardless?
```

Comment:
no, something else is supposed to set best-addr in that case.  at least, i think so

Reply:
dropped